src/PostgREST/Error.hs

```diff
@@ -250,7 +250,7 @@ data SchemaCacheError
   | NoRelBetween Text Text (Maybe Text) Text RelationshipsMap
   | NoRpc Text Text [Text] MediaType Bool [QualifiedIdentifier] [Routine]
   | ColumnNotFound Text Text
-  | TableNotFound Text Text [Table]
+  | TableNotFound Text Text SchemaCache
```

Comment:
```suggestion
  | TableNotFound Text Text Int TablesFuzzyIndex
```

Similar to https://github.com/PostgREST/postgrest/pull/4472#discussion_r2596117162.